sdk-base/src/main/java/com/mapbox/maps/extension/observable/model/RequestType.kt

```diff
@@ -4,53 +4,54 @@ import com.google.gson.annotations.SerializedName
 
 /**
  * Describes type for request object.
+ * @param value String value of this enum
  */
-enum class RequestType {
+enum class RequestType(val value: String) {
   /**
    * Request type unknown.
    */
   @SerializedName("unknown")
```

Comment:
Is SerializedName still used after Gson is removed?

Reply:
Please see Toburn's comment here: https://github.com/mapbox/mapbox-maps-android/pull/1001#discussion_r770599969

Reply:
in theory, developers could rely on it..

Reply:
=)